packages/jest-resolve/src/ModuleNotFoundError.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export default class ModuleNotFoundError extends Error {
```

Comment:
The Node 6 snapshot failures show pretty important change (not sure why it's different on higher versions though), since now we don't point to the place error happened, but to the error itself. We have a utility Error class called `ErrorWithStack` – maybe you could play with that?

Reply:
It looks like `jest-resolve` doesn't depend `jest-utils`, so can I use `Error.captureStackTrace` instead of extend ?

Reply:
If I can add dependency for `jest-resolve`, maybe we can move `ModuleNotFoundError` to `jest-util` as well

Reply:
Not sure about that, @SimenB?

Reply:
/cc @SimenB